### ISSUE TITLE
PhalconDriver assertion failures

### ISSUE THREAD

**Issue by cuewizchris** [BeatSwitch/lock#45]
Hi guys,

I'm trying to write a Phalcon driver for Lock, but testing it with the PersistentDriverTestCase you've provided, I get 10 failures. Since Phalcon's ORM isn't much different from Laravel's Eloquent, to which you've kindly provided a driver, I felt it should be a pretty easy job converting that driver to Phalcon, but I'm at a loss now.
Maybe these failures give you a clue what I've done wrong or where to look for the problems? The failures are as follows:

it_can_check_multiple_permissions_at_once
PersistentDriverTestCase.php:230

it_can_clear_multiple_permissions_at_once
PersistentDriverTestCase.php:242

it_only_clears_the_requested_permissions
PersistentDriverTestCase.php:255

it_only_clears_permissions_on_the_given_resource
PersistentDriverTestCase.php:268

it_can_clear_every_single_permission_for_a_lock_instance
PersistentDriverTestCase.php:282

it_can_toggle_multiple_permissions_at_once
PersistentDriverTestCase.php:329

it_can_check_actions_from_aliases
PersistentDriverTestCase.php:367

it_can_work_with_roles
PersistentDriverTestCase.php:402

it_can_return_allowed_resource_ids
PersistentDriverTestCase.php:452

it_can_return_denied_resource_ids
PersistentDriverTestCase.php:470

I'm grateful and happy about any feedback you can give me. Thank you for coding this very cool package.
Cheers, Christian

**Comment by cuewizchris** [comment]
Still getting three failures, but almost there.

**Comment by driesvints** [comment]
Ah cool. Sorry for not replying! I don't have much time to work on Lock these days. How did you solve them?

**Comment by cuewizchris** [comment]
Thank you for replying nonetheless! :D

I just used the new driver from beatswitch/lock-laravel (the one that uses one table to store permissions) instead of the one shown in the tutorial. That caused most of the problems I had to just magically disappear.

Nevertheless, there are still three failed assertions that I get, but I'm currently not sure if that's really caused by my driver.

In class PersistentDriverTestCase, there's line 68:
```
$lock->allow('create');
```
I think this causes the test `it_always_succeeds_with_the_all_action` to fail, particularly this assertion on line 174:
```
$this->assertFalse($lock->can('create', 'events'));
```
I'm not sure about this one. Is allowing the "create" action (on no particular resource, line 68) generally meant to permit "create" on all resources (line 174)? What's the intended behavior here, is my driver causing the trouble or line 174? :)
